apps/aecore/lib/aecore/tx/pool/worker.ex

```diff
@@ -152,6 +157,21 @@ defmodule Aecore.Tx.Pool.Worker do
 
       %OracleExtendTx{} ->
         tx.data.fee >= OracleExtendTx.calculate_minimum_fee(tx.data.payload.ttl)
+
+      %NameClaimTx{} ->
+        NameClaimTx.is_minimum_fee_met?(tx)
```

Comment:
Can't we just make a default variable `_` here and catch all the Txs where we just call `is_minimum_fee_met?()`?

Reply:
Prefer to have this explicit again, to not forget about it for new transactions. Whats your opinion about it?